src/Http/samples/MinimalSampleFSharp/Program.fs

```diff
@@ -0,0 +1,20 @@
+open System
+open Microsoft.AspNetCore.Builder
+open Microsoft.Extensions.Hosting
+
+[<EntryPoint>]
+let main args =
+    let app = WebApplication.Create(args);
+
+    if app.Environment.IsDevelopment() then
+        app.UseDeveloperExceptionPage() |> ignore
+
+    app.MapGet("/", Func<string>(fun () -> "Hello World!")) |> ignore
```

Comment:
Is the explicit `Func` construction needed here? Might be able to elide that

Reply:
It seems like it. Without it, I see the following error:

```
C:\dev\dotnet\aspnetcore\src\Http\samples\MinimalSampleFSharp\Program.fs(12,34): error FS0001: This expression was expected to have type↔    'AspNetCore.Http.HttpContext'    ↔but here has type↔    'unit' [C:\dev\dotnet\aspnetcore\src\Http\samples\MinimalSampleFSharp\MinimalSampleFSharp.fsproj]
```

Reply:
This is the F# language feature we want